R/app_server.R

```diff
@@ -1,12 +1,18 @@
-server = function(input, output, session) {
-
+#' Shiny application server
```

Comment:
This is mostly copied from server.R with a couple of changes.
THe main issue I noted was that some non-portable filepaths were used. There was a "inst/" prefix on some filepaths, and this meant that if the user wasn't in the package root when they call `run_app()` then the app didn't work.
This is easily fixed using system.file().

But, there are a few system.file calls in the file already, so it might be better to have a `dirs$data <- system.file("app", "www", "data", package = "tfpbrowser")` at the start of the server function, and build all subpaths on top of this using `file.path(dirs$data, "scanner_output")` etc.

Changing to use of system.file will break one of the code blocks - but it was a pretty fragile piece of code anyway. When constructing the path of one file relative to another, `fs::path_rel` is your friend.